src/patterns/components/promo/react/info/default.hbs

```diff
@@ -0,0 +1,147 @@
+      <td>string</td>
+      <td>
+        The text to include in the Call-to-Action.
+      </td>
+    </tr>
+    <tr>
+      <td class="sprk-u-FontWeight--bold">ctaHref</td>
+      <td>string</td>
+      <td>
+        The URL to navigate to when the Call-to-Action is clicked.
+      </td>
+    </tr>
+    <tr>
+      <td class="sprk-u-FontWeight--bold">ctaAnalytics</td>
+      <td>string</td>
+      <td>
+        The string to use for the data-analytics attribute on the Call-to-Action.
+      </td>
+    </tr>
+    <tr>
+      <td class="sprk-u-FontWeight--bold">ctaIdString</td>
+      <td>string</td>
+      <td>
+        The string to use for the data-id attribute on the Call-to-Action.
+      </td>
+    </tr>
+    <tr>
+      <td class="sprk-u-FontWeight--bold">imgSrc</td>
+      <td>string</td>
+      <td>
+        The source location of the Promo image. If this propterty is omitted, no image will be rendered.
+      </td>
+    </tr>
+    <tr>
+      <td class="sprk-u-FontWeight--bold">imgAlt</td>
+      <td>string</td>
+      <td>
+        The alt text property for the image.
+      </td>
+    </tr>
+    <tr>
+      <td class="sprk-u-FontWeight--bold">imgLinkHref</td>
+      <td>string</td>
+      <td>
+        The URL to navigate to when the image is clicked.
+      </td>
+    </tr>
+    <tr>
+      <td class="sprk-u-FontWeight--bold">imgLinkAnalytics</td>
+      <td>string</td>
+      <td>
+        The string to use for the data-analytics attribute on the image link.
+      </td>
+    </tr>
+    <tr>
+      <td class="sprk-u-FontWeight--bold">imgLinkIdString</td>
+      <td>string</td>
+      <td>
+        The string to use for the data-id attribute on the image link.
+      </td>
+    </tr>
+    <tr>
+      <td class="sprk-u-FontWeight--bold">additionalClassesImgLink</td>
+      <td>string</td>
+      <td>
+        Any additional classes (space-delimited string) to apply to the image link.
+      </td>
+    </tr>
+    <tr>
+      <td class="sprk-u-FontWeight--bold">idString</td>
+      <td>string</td>
+      <td>The string to use for the data-id property on the root Promo element.</td>
+    </tr>
+    <tr>
+      <td class="sprk-u-FontWeight--bold">isFlag</td>
+      <td>boolean</td>
+      <td>
+        If provided, the Promo will render as the Flag variant.
+      </td>
+    </tr>
+    <tr>
+      <td class="sprk-u-FontWeight--bold">mediaRev</td>
+      <td>boolean</td>
+      <td>
+        If provided, the Promo will be rendered in reverse horizontal order (content on the left, image on the right).
+      </td>
+    </tr>
+    <tr>
+      <td class="sprk-u-FontWeight--bold">hasBorder</td>
+      <td>boolean</td>
+      <td>
+        If true, the Promo will be rendered with a border. The styles for the border
+        are determined by <span class="sprk-u-FontWeight--bold">$sprk-promo-border</span>
+        and can be overridden in <span class="sprk-u-FontWeight--bold">_settings.scss</span>.
```

Comment:
@bobwise, The styles can't be overridden in _settings.scss. They would need to create their own Sass file and import it before the import of Spark styles. I think this should be rewritten to avoid confusion.